cartography/intel/gcp/sql.py

```diff
@@ -39,6 +39,8 @@ def get_sql_instances(sql: Resource, project_id: str, regions: list, common_job_
             if response.get('items', []):
                 for item in response['items']:
                     item['id'] = f"projects/{project_id}/instances/{item['name']}"
+                    for network in item.get('settings', {}).get('ipConfiguration', {}).get('authorizedNetworks',[]):
+                        item['authorizedNetworksList'].append(network['value'])
```

Comment:
get `authorizedNetworks ` object like this 
`item[authorizedNetworksList]=item.get('settings', {}).get('ipConfiguration', {}).get('authorizedNetworks',[])`